cinema/views.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):

```

Comment:
The `if` condition is not necessary here because `raise_exception=True` will raise an exception if the serializer is not valid. You can simply call `serializer.is_valid(raise_exception=True)` without the `if` condition. This is a code style issue according to the checklist.